Resources/Prototypes/Tiles/floors.yml

```diff
@@ -1181,3 +1209,23 @@
     collection: BarestepWood
   itemDrop: MaterialWoodPlank1
   heatCapacity: 10000
+
+#Hull tiles
+- type: tile
+  id: FloorHull
+  name: tiles-hull
+  sprite: /Textures/Tiles/hull.png
+  baseTurf: Plating
+  isSubfloor: false
+  canCrowbar: false
+  canAxe: false #You can't RCD these in SS13 apparently, so this is probably the next best thing
```

Comment:
im pretty sure i added a thing for rcd blacklist

Reply:
The component appears to be named "indestructible," which is definitely not what I want here (bombs should still probably be able to destroy this.) Unless the component doesn't actually make it indestructible...

Reply:
Indestructible is just for rcd as per the summary, might be worth renaming for clarity idk